test/tests/auxkernels/parsed_vector_aux/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  design = 'source/auxkernels/ParsedAux.md'
+  [test]
+    type = 'Exodiff'
+    input = 'parsed_aux_test.i'
+    exodiff = 'out.e'
+    issues = '#4807'
+    requirement = "The system shall be able to compute vector values for vector from a functional "
+                  "expression."
```

Comment:
"compute vector values for vector" is unclear to me

Reply:
reworded